splunk-cloudwatch-logs-processor/index.js

```diff
@@ -112,6 +113,11 @@ const getSplunkLogger = (parsed, context, callback) => {
 
             console.log('Data retrieved from SSM:', JSON.stringify(ssmData, null, 2));
 
+            // Make splunk_json optional
+            if (InvalidParameters in ssmData && ssmData.InvalidParameters.indexOf('splunk_json') != -1) {
+                ssmData.InvalidParameters.splice(ssmData.InvalidParameters.indexOf('splunk_json'), 1);
+            }
+
```

Comment:
@mpitcel any refactor suggestions to make this code clearer for folks who are not JS  coders?